src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/FlyoutTests/UnoSamples_Tests.Flyout.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.FlyoutTests
+{
+	[TestFixture]
+	public partial class Flyout_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[Ignore("Not available yet")]
```

Comment:
this might work on other platforms than Android. Can you test on WASM?

Reply:
sure thing

Reply:
tried but still couldnt access it